src/AbstractPluginManagerFactory.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Dash
+ *
+ * @link      http://github.com/DASPRiD/Dash For the canonical source repository
+ * @copyright 2013-2015 Ben Scholzen 'DASPRiD'
+ * @license   http://opensource.org/licenses/BSD-2-Clause Simplified BSD License
+ */
+
+namespace Dash;
+
+use Interop\Container\ContainerInterface;
+use Zend\ServiceManager\AbstractPluginManager;
+use Zend\ServiceManager\Factory\FactoryInterface;
+
+/**
+ * Abstract factory for plugin managers.
+ */
+abstract class AbstractPluginManagerFactory implements FactoryInterface
+{
+    /**
+     * @var string
+     */
+    protected $configKey;
+
+    /**
+     * @var string
+     */
+    protected $className;
```

Comment:
I suppose that's valid. I gotta see how much two additional function calls would impact the performance, but probably they don't too much.